semgrep-core/tests/tainting_rules/php/opaque_safe.yaml

```diff
@@ -0,0 +1,14 @@
+rules:
+  - id: tainted
+    languages:
+      - php
+    message: Match
+    mode: taint
+    pattern-sanitizers:
+      - pattern: sanitizer(...)
```

Comment:
should it be $F instead here?
to match the example mentioned in the PR?

Reply:
That's right I forgot to update it! Test fails otherwise. Fixing it.